src/java/fr/paris/lutece/plugins/forms/web/admin/FormJspBean.java

```diff
@@ -456,6 +472,199 @@ public String getModifyForm( HttpServletRequest request ) throws AccessDeniedExc
 
         return redirectView( request, VIEW_MANAGE_FORMS );
     }
+    
+    @View( VIEW_MANAGE_EXPORT )
+    public String getManageExport( HttpServletRequest request ) throws AccessDeniedException
+    {
+        int nId = NumberUtils.toInt( request.getParameter( FormsConstants.PARAMETER_ID_FORM ), FormsConstants.DEFAULT_ID_VALUE );
+
+        if ( nId == FormsConstants.DEFAULT_ID_VALUE )
+        {
+            return redirectView( request, VIEW_MANAGE_FORMS );
+        }
+        
+        checkUserPermission( Form.RESOURCE_TYPE, String.valueOf( nId ), FormsResourceIdService.PERMISSION_MODIFY_PARAMS, request );
+
+        Form formToBeModified = FormHome.findByPrimaryKey( nId );
+
+        if ( formToBeModified == null )
+        {
+            return redirectView( request, VIEW_MANAGE_FORMS );
+        }
+        
+        Map<String, Object> model = getModel( );
+        model.put( MARK_FORM, formToBeModified );
+        model.put( MARK_EXPORT_LIST,  ExportServiceManager.getInstance( ).createReferenceListExportConfigOption( formToBeModified, getLocale( ) ) );
+        model.put( MARK_EXPORT_CONFIG_LIST, ExportServiceManager.getInstance( ).createReferenceListExportConfig( formToBeModified, getLocale( ) ) );
+        
+        return getPage( PROPERTY_PAGE_TITLE_MODIFY_FORM, TEMPLATE_MANAGE_EXPORT, model );
+    }
+    
+    @Action( ACTION_CREATE_EXPORT_CONFIG )
+    public String doCreateExportConfig( HttpServletRequest request )
+    {
```

Comment:
Maybe you should add CSRF protections ?

Reply:
As it seems that no jspbean integrates CSRF protections, it would be better to make these changes in a separate commit for the whole forms plugin.